cli.mjs

```diff
@@ -185,19 +242,19 @@ async function deploymentPrompt() {
 }
 
 /** Prompt the user to enter the ID of a study */
-async function studyIDPrompt() {
+// helper to check if the given study (input) is in firestore
+const validateStudyFirebase = async (input) => {
   const invalidMessage = "Please enter a valid study from your Firestore database";
-  const validateStudyFirebase = async (input) => {
-    // subcollection is programmatically generated, if it doesn't exist then input must not be a valid studyID
-    const studyIDCollections = await getStudyRef(input).listCollections();
-    return studyIDCollections.find((c) => c.id === PARTICIPANTS_COL) ? true : invalidMessage;
-  };
+  // subcollection is programmatically generated, if it doesn't exist then input must not be a valid studyID
+  const studyIDCollections = await getStudyRef(input).listCollections();
+  return studyIDCollections.find((c) => c.id === PARTICIPANTS_COL) ? true : invalidMessage;
```

Comment:
Hmm... I know this is code I wrote but it looks a little funky to me. Are you okay changing it here?

I would return `true` or `false` from this function and then in `main` where you have that `console.error` I would just call the string itself - `console.error("Please enter a valid study from your Firestore database")`